db/migrate/20220225074245_create_users.rb

```diff
@@ -4,7 +4,7 @@ def change
       t.string :first_name
       t.string :last_name
       t.string :username
-      t.string :password_hash
+      t.string :password_digest
```

Comment:
Don't rewrite migrations in separate commits. If you're changing the migration, it needs to be fixed up in the original commit or otherwise you can create conflicts in the database.

Reply:
If you do need to change a column, make a new migration:
```sh
rails g migration rename_password_hash_to_digest_on_users
```
```ruby
def change
  rename_column :users, :password_hash, :password_digest
end
```

Reply:
Yeah apologises on that; I know in a real-world scenario it is extremely bad foo due to existing databases; here I simply altered due to not having any data as of yet, however I'll keep that in mind and not take any short cuts in regards to migrations even on empty databases, thanks Matt!